eslint.config.js

```diff
@@ -0,0 +1,31 @@
+import js from '@eslint/js';
+import globals from 'globals';
+
+export default [
+  {
+    ignores: [
+      'src/test/webapp/**/*',
+      'dist',
+      'build',
+    ],
+  },
+  js.configs.recommended,
+  {
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        QUnit: true,
+        tests: true,
+      },
+    },
+    rules: {
+      'no-var': 'error',
+      'prefer-const': 'error',
+      'quotes': ['error', 'single', { 'avoidEscape': true, 'allowTemplateLiterals': true }],
```

Comment:
```suggestion
      'quotes': ['error', 'double', { 'avoidEscape': true, 'allowTemplateLiterals': true }],
```

For consistency with the current codebase.